doc/windows/package-manager/package/winget-validation.md

```diff
@@ -54,14 +54,14 @@ immediately. Some may trigger manual validation.
 |--------------|-------------|
 |<img width=200/>|<img width=500/>|
 | **Binary-Validation-Error** | The application included in this **Pull Request** failed to pass the **Installers Scan** test. This test is designed to ensure that the application installs on all environments without warningsFor further details on this error, see [binary validation errors](.\b. inary-validation-errors.md). |
```

Comment:
Pretty sure you want `binary` not `\b.inary` here...

Reply:
Yup thanks